test/test_optimize.py

```diff
@@ -28,6 +29,20 @@ def mode(request):
     ('dlib', ''),
     ('pyswarm', ''),
     ('cmaes', ''),
+    *[('nlopt', method) for method in [
```

Comment:
wow, testing all :laughing: but yes, makes sense.

Reply:
with the rosen objective these finish in a matter of a couple of seconds, makes me more confident they are not completely terrible :D